### ISSUE TITLE
PrimoCache is ZFS for Windows!??

### ISSUE THREAD

**Issue by Logic-Elliven** [openzfsonwindows/ZFSin#184]
When you look at the caching behavior of PrimoCache it is eerily similar to that of ZFS.
ie:  It is possible that Primocache is in fact a ZFS windows driver with a fancy GUI and all!?
ie:  I'm no dev but perhaps downloading their trial version and scratching around in the files may be very illuminating?

https://www.romexsoftware.com/en-us/primo-cache/

**Comment by lundman** [comment]
no, they are a cacher. It is true ZFS also has a a few cache components, ARC, l2arc, slog, but it is also a storage pool environment, and posix filesystem.

**Comment by the-insulter** [comment]
> 
> 
> When you look at the caching behavior of PrimoCache it is eerily similar to that of ZFS.
> ie: It is possible that Primocache is in fact a ZFS windows driver with a fancy GUI and all!?
> ie: I'm no dev but perhaps downloading their trial version and scratching around in the files may be very illuminating?
> 
> https://www.romexsoftware.com/en-us/primo-cache/

Hi,

absolutely two different things.

Primocache is what smartdrive was in the old good days of HDD-Caching.
It is Borrowed RAM from System which the system also can/(should) not affect with, cause here are cached data for HD/SSD to be written or read before accessing HD/SSD again, saving I/O and energy and stupidness of developers and programmers and a lot of more.

It fucks you the moment there is no energy and all your important data is not written to disk, but hey, better that than get fucked by Trump or ???

ZFS has some Buffering (low amount of borrowed OS-Ram to do it's job.

By the way, Primocache would also be good for linux, as YOU as the USER, can affect its behavior and are not fucked-arsed buy Linus Torvalds and the kind smart freaks who think their programmed in code is the best for every one to eat... (or so, as by getting fucked between Linux or Trump I would still prefer Linux, he has some smartness ... !!! 8:\ )

Bottom line:
something like Primocache and ZFS (I prefer XFS) fpr Windows and Linux would make humanity jumpstart in computer science about 40 light years ....
